### DISCUSSION TITLE
Provide the application insights powershell script

### DISCUSSION THREAD

**Discussion by CurlyBytes** [WorkSafeBC-Common-Engineering/Common.Engineering.Discussion#17] [General]
Hello everyone, im really curious the APM tools in the devops platform to track the activity with, can you share the powershell script that the team is using? thanks!

**Comment by CurlyBytes** [comment]
my bad, i did not expect the application insights repo , i just look on this repo https://github.com/WorkSafeBC-Common-Engineering/AzureDevOps.Automation.Pipeline.Templates.v2

i just later found out it was on this repo https://github.com/WorkSafeBC-Common-Engineering/AzureDevOps.Automation.Pipeline.Toolbox.v2/blob/master/toolbox/scripts/application-insights/log-bootstrap-event-to-ai.ps1 , thanks!

**Reply by wpschaub** [reply]
No problem. Thank you for letting us know you found it - perhaps you have ideas how we can improve our documentation?
